libraries/MySensors/MySensor.cpp

```diff
@@ -102,7 +102,7 @@ void MySensor::begin(void (*_msgCallback)(const MyMessage &), uint8_t _nodeId, b
 
 	// Setup radio
 	if (!radio.init()) {
-		debug(PSTR("radio initialization failed!"));
+		debug(PSTR("radio initialization failed!\n"));
```

Comment:
Right. I was thinking that it may be reasonable to try the radio initialization again if previously failed (in the loop()). E.g. I have board Devduino v2.0 on which I have to unplug the RF24 module when programming the Atmega. So once the board is reset after the programming I may plug in the RF24 late, so another initialization may succeed.

Other than this situation, the failed initialization is commonly due to bad wiring so there is rally not much sense in executing any other code.

So it depends if these extra lines of code and extra program memory used by it is worth it.